app/app/vhosts/index/route.js

```diff
@@ -9,5 +9,25 @@ export default Ember.Route.extend({
     if(model.get('length') === 0) {
       return this.transitionTo('app.vhosts.new', this.modelFor('app'));
     }
+  },
+
+  actions: {
+    delete: function(vhost){
+      let op = this.store.createRecord('operation', {
+        type: 'deprovision',
+        vhost: vhost
+      });
+
+      this.controller.set('error', null);
+      vhost.set('isDeleting', true);
```

Comment:
as of 027b883ca11123852e9a22f66a20208ac68a591c we have a store on the components, also